### DISCUSSION TITLE
500 Face Limit in Web Browser and Mobile App

### DISCUSSION THREAD

**Discussion by trop1kal** [immich-app/immich#13559] [Q&A] [answered]
Just upgraded from 117 to 118 and noticed all the unnamed faces are now missing.
I've tried toggling back and fourth which makes no change.

Only change I made when upgrading was the Port alignment.
start.sh arguments were already removed or never excited in my config file.

Any help or anything I can try to restore un-named faces so I can continue tagging people.

Also noticed when I change a persons "Feature Photo" the picture doesn't change.

**_This is affecting both the Android App and Web/Desktop Browser version._**

Any help would be greatly appreciated.

**Edit**:
Looking further and finding random named faces are also missing. I don't see the user's in the list of faces nor can I search for them any longer.
The people count in the top left hand corner reflects "People (2,359)" which is correct but theres no way 2,359 faces are being displayed. I used to have to scroll 2-3x to reach the bottom.

**2nd Edit**:
Looking further what it seems like is happening is its only displaying 500 faces. Theres nothing to switch to another page or anything along the lines.

Is there a change with 118 that limits faces shown to 500?

**Comment by trop1kal** [comment]
Hi Devs.. How do I go about tagging unnamed faces if I cant see them no longer due to this 500 people restriction within the web browser and mobile app?

Is there another method I'm just missing?

This only started happening when I upgraded to 118.2

In addition when the faces don't show in the list I can't search for the person. Its like they dont exist even though I know they are there because I just tagged there face before doing this update.

**Comment by trop1kal** [comment]
Anyone know how to bypass this 500 face limit?

**Reply by alextran1502** [reply, marked as answer]
Fixed, will be available in the next release

**Reply by trop1kal** [reply]
Awesome! TY!
